docs/database-classes.md

```diff
@@ -279,6 +347,12 @@ transactor.Transact(db =>
 
 ## Limitations
 
+* All database classes have to be declared as `public abstract`.
+* Database classes may not inherit non-database classes.
+* Database classes shall have either no declared constructor, or a public parameterless constructor.
+* All database properties have to be declared as `public abstract`, with one of the supported [database types](database-types.md) as property type.
```

Comment:
* Database properties must be declared as `public abstract`, with one of the supported [database types](database-types.md) as property type.